harpy/commands/report.py

```
@@ -43,13 +45,13 @@ def report(directory, debug, headless, clear_cache, port, server_port, refresh):
     if os.path.isdir("_build") and clear_cache:
         rmtree("_build", ignore_errors=True)
 
-    tracker = ReportRender(directory if directory else "")
+    tracker = ReportRender(directory if directory else "", md)
     tracker.scan_for_reports()
     tracker.update_yaml()
     URL = ""
     myst_error = ""
     if debug:
-        os.system(" ".join(cmd))
+        subprocess.run(cmd, stdout = sys.stdout, stderr = sys.stderr)
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)